app/uk/gov/hmrc/vatregisteredcompaniesstub/connectors/BackendConnector.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompaniesstub.connectors
+
+//import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import javax.inject.Singleton
+import play.api.libs.json.Writes
+import uk.gov.hmrc.http.hooks.HttpHook
+import uk.gov.hmrc.http.logging.Authorization
+import uk.gov.hmrc.http.{HeaderCarrier, HttpPost, HttpReads}
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.http.ws.WSPost
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+trait BackendConnector extends ServicesConfig {
```

Comment:
consider making this an object (or maybe a package object)
Assuming it can be separates from ServicesConfig

Reply:
Services config is providing access to the baseUrl and token.